src/rbr/RBR.java

```diff
@@ -312,18 +312,18 @@ private void merge(Vertex router) {
      */
     private boolean mergeUnitary(Vertex router) {
         for (int a = 0; a < regionsForVertex.get(router).size(); a++) {
-            Region ra = regionsForVertex.get(router).get(a);
-            for (int b = a + 1; b < regionsForVertex.get(router).size(); b++) {
-                Region rb = regionsForVertex.get(router).get(b);
-                if (ra.canBeMergedWith(rb)) {
-                    Region reg = ra.merge(rb);
-                    regionsForVertex.get(router).add(reg);
-                    regionsForVertex.get(router).remove(ra);
-                    regionsForVertex.get(router).remove(rb);
-                    return true;
-                }
-            }
-        }
-        return false;
+             Region ra = regionsForVertex.get(router).get(a);
+             for (int b = a + 1; b < regionsForVertex.get(router).size(); b++) {
+                 Region rb = regionsForVertex.get(router).get(b);
+                 if (ra.canBeMergedWith(rb)) {
+                     Region reg = ra.merge(rb);
+                     regionsForVertex.get(router).add(reg);
+                     regionsForVertex.get(router).remove(ra);
+                     regionsForVertex.get(router).remove(rb);
+                     return true;
+                 }
+             }
+         }
+         return false;
```

Comment:
Each tab should have four spaces, in above lines (315 - 327) there is two tabs and an extra space. Please, remove this extra space.